python/observability-online-eval/livekit/livekit-gemini.py

```
@@ -6,6 +6,7 @@
 from livekit import agents
 from livekit import api as livekit_api
 from livekit.agents import Agent, AgentSession, function_tool
+from livekit.agents.utils import images
```

Pylint (3.3.7)

[error] Unable to import 'livekit.agents.utils'

(E0401)